core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -269,7 +269,7 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("metrics directory")
       .ofType(classOf[java.lang.String])
-    val newConsumerOpt = parser.accepts("new-consumer", "Use the new consumer implementation. This is the default.")
+    val newConsumerOpt = parser.accepts("new-consumer", "This is the default, so this option is deprecated and will be removed in a future release.")
```

Comment:
We should still have the description of the parameter. I'll add it back before merging.